src/pages/benefit.js

```diff
@@ -70,6 +70,13 @@ export function BenefitPage() {
     // if not logged in log in first
     if (!keycloak.authenticated) {
       keycloak.login();
+    } else if (benefitData.benefitId === "olic") {
+      window.open(
+        "https://www.ontario.ca/page/low-income-workers-tax-credit",
+        "_blank"
+      );
+    } else if (benefitData.benefitId === "occs") {
+      window.open("https://www.ontario.ca/page/child-care-subsidies", "_blank");
```

Comment:
There is a `redirect_url` that comes from the benefit service. You would need to modify the benefit selector to access the variable, but that would mean you don't have to hard code the URLs here, as well as only needing one `else if`.

Reply:
I modified it now.